[2023-03-07T18:19:07.624+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Ingest_to_Snowflake.copy_into_races_table manual__2023-03-07T18:19:00.051268+00:00 [queued]>
[2023-03-07T18:19:07.631+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Ingest_to_Snowflake.copy_into_races_table manual__2023-03-07T18:19:00.051268+00:00 [queued]>
[2023-03-07T18:19:07.632+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-07T18:19:07.632+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-07T18:19:07.632+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-07T18:19:07.784+0000] {taskinstance.py:1300} INFO - Executing <Task(S3ToSnowflakeOperator): copy_into_races_table> on 2023-03-07 18:19:00.051268+00:00
[2023-03-07T18:19:07.789+0000] {standard_task_runner.py:55} INFO - Started process 8799 to run task
[2023-03-07T18:19:07.791+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Ingest_to_Snowflake', 'copy_into_races_table', 'manual__2023-03-07T18:19:00.051268+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/copy_into_snowflake.py', '--cfg-path', '/tmp/tmppu8vavsb']
[2023-03-07T18:19:07.792+0000] {standard_task_runner.py:83} INFO - Job 111: Subtask copy_into_races_table
[2023-03-07T18:19:07.838+0000] {task_command.py:388} INFO - Running <TaskInstance: Ingest_to_Snowflake.copy_into_races_table manual__2023-03-07T18:19:00.051268+00:00 [running]> on host d7fd3210ba1e
[2023-03-07T18:19:07.888+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Ingest_to_Snowflake
AIRFLOW_CTX_TASK_ID=copy_into_races_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-07T18:19:00.051268+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-07T18:19:00.051268+00:00
[2023-03-07T18:19:07.888+0000] {s3_to_snowflake.py:146} INFO - Executing COPY command...
[2023-03-07T18:19:07.895+0000] {base.py:73} INFO - Using connection ID 'snowflake_conn' for task execution.
[2023-03-07T18:19:07.897+0000] {connection.py:287} INFO - Snowflake Connector for Python Version: 3.0.1, Python Version: 3.7.16, Platform: Linux-5.15.79.1-microsoft-standard-WSL2-x86_64-with-debian-11.6
[2023-03-07T18:19:07.898+0000] {connection.py:990} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-03-07T18:19:07.898+0000] {connection.py:1007} INFO - Setting use_openssl_only mode to False
[2023-03-07T18:19:08.376+0000] {cursor.py:738} INFO - query: [ALTER SESSION SET autocommit=True]
[2023-03-07T18:19:08.447+0000] {cursor.py:751} INFO - query execution done
[2023-03-07T18:19:08.447+0000] {cursor.py:891} INFO - Number of results in first chunk: 1
[2023-03-07T18:19:08.447+0000] {sql.py:375} INFO - Running statement: COPY INTO PUBLIC.races
FROM @formula_1_stage/temi-formula1/
files=('races.csv')
file_format=generic_csv, parameters: None
[2023-03-07T18:19:08.448+0000] {cursor.py:738} INFO - query: [COPY INTO PUBLIC.races FROM @formula_1_stage/temi-formula1/ files=('races.csv') ...]
[2023-03-07T18:19:09.466+0000] {cursor.py:751} INFO - query execution done
[2023-03-07T18:19:09.471+0000] {connection.py:586} INFO - closed
[2023-03-07T18:19:09.493+0000] {connection.py:589} INFO - No async queries seem to be running, deleting session
[2023-03-07T18:19:09.562+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/snowflake/transfers/s3_to_snowflake.py", line 147, in execute
    snowflake_hook.run(copy_query, self.autocommit)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 379, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
  File "/home/airflow/.local/lib/python3.7/site-packages/snowflake/connector/cursor.py", line 839, in execute
    Error.errorhandler_wrapper(self.connection, self, error_class, errvalue)
  File "/home/airflow/.local/lib/python3.7/site-packages/snowflake/connector/errors.py", line 294, in errorhandler_wrapper
    error_value,
  File "/home/airflow/.local/lib/python3.7/site-packages/snowflake/connector/errors.py", line 345, in hand_to_other_handler
    cursor.errorhandler(connection, cursor, error_class, error_value)
  File "/home/airflow/.local/lib/python3.7/site-packages/snowflake/connector/errors.py", line 231, in default_errorhandler
    cursor=cursor,
snowflake.connector.errors.ProgrammingError: 091016 (22000): 01aacaab-0000-0ab4-0000-1e09000140ee: Remote file 's3://temi-formula1/temi-formula1/races.csv' was not found. If you are running a copy command, please make sure files are not deleted when they are being loaded or files are not being loaded into two different tables concurrently with auto purge option.
[2023-03-07T18:19:09.569+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=Ingest_to_Snowflake, task_id=copy_into_races_table, execution_date=20230307T181900, start_date=20230307T181907, end_date=20230307T181909
[2023-03-07T18:19:09.588+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 111 for task copy_into_races_table (091016 (22000): 01aacaab-0000-0ab4-0000-1e09000140ee: Remote file 's3://temi-formula1/temi-formula1/races.csv' was not found. If you are running a copy command, please make sure files are not deleted when they are being loaded or files are not being loaded into two different tables concurrently with auto purge option.; 8799)
[2023-03-07T18:19:09.606+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-07T18:19:09.628+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
